include/deal.II/fe/fe_system.h

```diff
@@ -475,7 +475,11 @@ class FESystem : public FiniteElement<dim, spacedim>
    * In other words, if no multiplicity for an element is explicitly specified
    * via the exponentiation operation, then it is assumed to be one (as one
    * would have expected).
+   *
+   * @warning This feature is not available for Intel compilers
+   * prior to version 19.0
    */
+#  if !defined(__INTEL_COMPILER) || __INTEL_COMPILER >= 1900
```

Comment:
I don’t have issues with 18.02

Reply:
Please lower the requirement to 18

Reply:
So you can run `fe/system_03` and `fe/system_04` successfully?

Reply:
I did not run any tests (that’s a local cluster, admins wouldn’t appreciate it I think), but compilation is ok.

Reply:
Yes, of course, I was testing with ICC-18 regularly, but these tests always failed. Hence, I want to document that people can't expect these constructors to work before ICC-19 (for which the tests are indeed passing). Having a different restriction in the source code seems to be inconsistent and the result is the same. The compiler never considered these constructors in overload resolution properly.

Reply:
Thanks for explanation